requirements.txt

```diff
@@ -5,11 +5,11 @@
 pbr~=5.4 # Apache-2.0
 aiohttp~=3.6
 appdirs~=1.4
-webargs<6,>=5
-marshmallow<3.0.0rc6,>=3.0.0rc4
+webargs~=5.5
+marshmallow~=3.2
 numpy~=1.7
 Cython~=0.29
-apispec<1,>=0.21
+apispec[yaml,validation]~=3.1
```

Comment:
Could we do the same for our test / dev packages in `setup.py`

Reply:
Do you mean the `test-requirements.txt`? We don't define any packages in setup.py.

We do need to re-arrange what goes in requirements.txt and test-requirements.txt (and create, I think, a `build-requirements.txt`). #229 describes the issue a bit.

Reply:
Something like `extras_require=dict(test=get_requirements_dev()...)` <- from arviz https://github.com/arviz-devs/arviz/blob/master/setup.py

It is currently commented out, because arviz have github dependencies.